_posts/2021-10-29-moving-from-opensource-elasticsearch-to-opensearch.markdown

```diff
@@ -50,8 +51,11 @@ POST /_reindex
    }
 }
 ```
+Lastly, download the latest version of OpenSearch from the [OpenSearch downloads page](https://opensearch.org/downloads.html). If you need help with installation, see the guidance in [Install and Configure OpenSearch](https://opensearch.org/docs/latest/opensearch/install/index/).
+
+As described in a [previous blog post](https://opensearch.org/blog/technical-posts/2021/07/how-to-upgrade-from-opendistro-to-opensearch/), open-source Elasticsearch is the foundation for Open Distro, and it has the same upgrade paths. You can follow through the steps for the path you choose to upgrade your cluster.
```

Comment:
@saratvemulapalli Is it still relevant to link to this blog? Is there an updated resource we want to include instead?

Reply:
@vagimeli definitely. Looking at the blog it goes in details of how each type of upgrade should be performed irrelevant of versions/Elasticsearch/OpenSearch.